builder/src/main/java/dev/sashimono/builder/jar/JarTask.java

```diff
@@ -100,9 +101,22 @@ private void writeManifestFile(final Path classesDir) throws IOException {
         Files.createDirectories(metaInfDir);
         try (final BufferedWriter writer = Files.newBufferedWriter(manifestPath)) {
             final String delimiter = ": ";
-            for (Map.Entry<String, String> entry : manifestEntries.entrySet()) {
-                writer.write(StringUtil.camelToCapitalisedKebabCase(entry.getKey()) + delimiter + entry.getValue()
-                        + System.lineSeparator());
+            final int max_line_bytes = 72;
+            final int max_line_bytes_less_eol = max_line_bytes - 2;
+            for (final Map.Entry<String, String> entry : manifestEntries.entrySet()) {
+                String line = StringUtil.camelToCapitalisedKebabCase(entry.getKey()) + delimiter + entry.getValue();
+                final StringBuilder toWrite = new StringBuilder();
+                while (line.getBytes().length > max_line_bytes) {
+                    int bytes = max_line_bytes_less_eol;
+                    String newLine = line.substring(0, bytes);
+                    while (newLine.getBytes().length > max_line_bytes && bytes > 0) {
```

Comment:
What is this while loop for? newLine is always 70 bytes long, and max_line_bytes is 72, so I can't see how it is ever executed?

Reply:
Oh, I guess this is for multibyte characters

Reply:
You should be using getBytes(StandardCharsets.UTF8) though, we never want to depend on the system locale.